src/Util/Common.hs

```diff
@@ -49,4 +49,13 @@ updateCamera game = game
     ratio = minimum [ (fromIntegral . fst $ initialWindowDimensions) / (fst boundaryDimensions)
                     , (fromIntegral . snd $ initialWindowDimensions) / (snd boundaryDimensions)
                     , 1
-                    ]
\ No newline at end of file
+                    ]
+
+
+gameObjects :: Game -> [Object]
+gameObjects game = map playerObject (gamePlayers game)
+                   ++ levelObjects (gameLevel game)
+
+
+updateObjects :: Game -> Game
+updateObjects game = foldr (\obj -> (objectOnUpdate obj) obj) game (gameObjects game)
```

Comment:
Consider changing (objectOnUpdate obj) to objectOnUpdate obj $ ...

Reply:
done